spec/rubocop/cop/lint/literal_as_condition_spec.rb

```diff
@@ -138,13 +138,6 @@
       RUBY
       expect(cop.offenses.size).to eq(1)
     end
-
-    it "registers an offense for `not #{lit}`" do
-      inspect_source(<<-RUBY.strip_indent)
-        !#{lit}
```

Comment:
We should not remove this test case, it should be `not(#{lit})`. Sorry, this test is my mistake.

Reply:
Fixed